### PR TITLE
fix: add patch for abseil on jetson platform

### PR DESCRIPTION
**Pull request by wep21**
- patch from https://github.com/abseil/abseil-cpp/commit/372124e6af36a540e74a2ec31d79d7297a831f98

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @derekmauro, modules you maintain (abseil-cpp) have been updated in this PR. Please review the changes.

File: modules/abseil-cpp/20240722.0.bcr.1/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://bazel.build/external/overview#bzlmod
+
+module(
+    name = "abseil-cpp",
+    version = "20240722.0.bcr.1",
+    compatibility_level = 1,
```
Comment (Wyverald) [inline modules/abseil-cpp/20240722.0.bcr.1/MODULE.bazel:20]: `bazel_compatibility = [">=7.2.1"],` for overlay usage

Comment (wep21) [inline modules/abseil-cpp/20240722.0.bcr.1/MODULE.bazel:20]:
```suggestion
    compatibility_level = 1,
    bazel_compatibility = [">=7.2.1"],
```

Comment (wep21) [reply modules/abseil-cpp/20240722.0.bcr.1/MODULE.bazel:20]: addressed https://github.com/bazelbuild/bazel-central-registry/pull/3196/commits/d5a4c13da460c46b3931bc83dbb5718248fdac88.

**Comment by Wyverald** [comment]
I'll let @derekmauro do the actual approval.

**Review by Wyverald — APPROVED** [review]
since Derek is on leave for another ~2 weeks, and this change looks innocent enough, I'll just go ahead and approve it.

**Pull request merged**